BDArmory/UI/BDArmoryAIGUI.cs

```diff
@@ -1193,7 +1199,7 @@ void WindowRectAI(int windowID)
                             ActivePilot.defaultAltitude =
                                 GUI.HorizontalSlider(SettingSliderRect(leftIndent, altLines, contentWidth),
                                     ActivePilot.defaultAltitude, 100, ActivePilot.UpToEleven ? 100000 : 15000);
-                            ActivePilot.defaultAltitude = Mathf.Round(ActivePilot.defaultAltitude / 25) * 25;
+                            ActivePilot.defaultAltitude = Mathf.Round(ActivePilot.defaultAltitude / 50) * 50;
```

Comment:
matching these to PAW slider increments

Reply:
The rounding here should actually match whatever level the slider resolution sets it to in `OnSliderResolutionUpdated`, which can also adjust the minimum. It's on my todo list from when I added that slider, but I haven't gotten around to it yet.
Also, more fields should be tweakable by the slider resolution, currently it's only PID, altitudes, speeds and the extend distances.